frontend/public/components/RBAC/bindings.jsx

```diff
@@ -317,8 +317,9 @@ export const RoleBindingsPage = ({
 
   const data = React.useMemo(() => flatten(resources), [resources]);
 
-  // Show table if at least a RBAC loaded property is true
-  const loaded = Object.values(resources).some((r) => r.loaded);
+  // Show table if at least a RBAC loaded property is true and ClusterRoleBinding is loaded (this is needed to ensure filtering is correct)
+  const loaded =
+    Object.values(resources).some((r) => r.loaded) && resources.ClusterRoleBinding.loaded;
```

Comment:
Won't this be the same as...?

```ts
  const loaded = resources.ClusterRoleBinding.loaded;
```

I'd probably just change it to

```suggestion
  const loaded = Object.values(resources).every((r) => r.loaded);
```

Reply:
🤦‍♀️   Sorry - fixed now